docs/getting_started/working_in_github.md

```diff
@@ -0,0 +1,48 @@
+# Getting the code
```

Comment:
This partially overlaps with https://github.com/project-chip/connectedhomeip/blob/master/CONTRIBUTING.md. Do we need both?

Reply:
I will remove it - I do not have the time to re-write the entire documentation section at this time.